File: Python/simple-array-sum.py
arrLen = int(input("Masukkan panjang array = "))


def arrSum(arrLen):
    array = [None] * arrLen

    total = 0
    for i in range(0, arrLen):
        array[i] = int(input("Elemen ke-" + str(i + 1) + "= "))
        total += array[i]

    return total


print(arrSum(arrLen))

File: Golang/simple-array-sum/simple-array-sum.go
package main

import "fmt"

func main() {
	var arrLen int
	fmt.Printf("Panjang array = ")
	fmt.Scan(&arrLen)

	var total = arrSum(arrLen)
	fmt.Printf("Jumlah elemen array adalah %v\n", total)
}

func arrSum(arrLen int) int {
	var arr = make([]int, arrLen)
	total := 0

	for i := 0; i < arrLen; i++ {
		fmt.Printf("Elemen ke-%v = ", i+1)
		fmt.Scan(&arr[i])
		total += arr[i]
	}
	return total
}
